app/src/main/java/com/example/drawingApp/MainActivity.kt

```diff
@@ -26,11 +24,45 @@ class MainActivity : AppCompatActivity() {
         drawingFieldView.onBitmapUpdate = {
             model.activeBitmap = it
         }
-
+        //when we click to create a new layer
         findViewById<TextView>(R.id.newLayer).setOnClickListener {
-            showLayerAlertDialog()
+            //assign the alert dialog to be able to dismiss later
```

Comment:
just gonna leave this review comment here to catch all the comments,

I would review and see what information you think is critical to understanding the reasoning for something, some of these comments are very helpful to knowing what is going on, and others are more of pseudocode for you to follow